### DISCUSSION TITLE
Compass Clue Solver

### DISCUSSION THREAD

**Discussion by pr-1996** [whs/runekit#26] [Q&A macOS]
Is there any known issue with the compass clue solver not being able to triangulate? I've tried Alt+1 combination (ie. option+1) as well as CMD+1 amongst other combinations and executing as on Windows, but no location input is gathered.

**Comment by schoentoon** [comment]
You can double click on the map to manually set a marker. After that click on the button which will have coordinates in it. This will create the line as you expect. It's not ideal, but it works. With the actual alt1 it would read out your screen and figure out what teleport you hovered over when pressing alt+1. Last I checked however this isn't implemented correctly in runekit.

**Reply by pr-1996** [reply]
Not ideal, but works great. Thank you for the suggestion 😀

**Comment by RRichardsDev** [comment]
I am having this same issue...
